src/engine/openspaceengine.cpp

```diff
@@ -869,7 +866,16 @@ void OpenSpaceEngine::loadSingleAsset(const std::string& assetPath) {
 
     _writeDocumentationTask = std::async(&OpenSpaceEngine::writeSceneDocumentation, this);
 
-    LTRACE("OpenSpaceEngine::loadSingleAsset(end)");
+    LTRACE("OpenSpaceEngine::loadAsset_init(end)");
+}
+
+void OpenSpaceEngine::loadAsset_postInit(const std::string assetName) {
```

Comment:
This function is (rightfully?) not called anymore and can die in that case

Reply:
Deleted this function